src/Files/Strings/da/Resources.resw

```diff
@@ -1686,4 +1791,10 @@
   <data name="InsertDiscDialog.Title" xml:space="preserve">
     <value>Indsæt en disk</value>
   </data>
+  <data name="Extract" xml:space="preserve">
+    <value>Udpak</value>
+  </data>
+  <data name="ExtractKeyboardAcceleratorTextOverride" xml:space="preserve">
+    <value>Ctrl+E</value>
+  </data>
```

Comment:
Can you revert the changes to the resw files? We usally generate them separately to make the PR review easier.